cmake/winml.cmake

```diff
@@ -101,7 +101,6 @@ target_cppwinrt(winml_api_experimental
   ${winml_midl_defines}                        # the midl compiler defines
   ${winml_api_use_ns_prefix}                   # set ns_prefix
 )
-add_dependencies(winml_api_experimental winml_api)
```

Comment:
Can you explain how to race condition happens?

Reply:
All I know is in the PR description. I dug a bit through `target_cppwinrt` in `cmake/winml_cppwinrt.cmake` to find out more but it's a wild mix of cmd/powershell/cmake that is hard to reason about, we should probably rewrite the entire thing :( . I'll investigate some more